chatbot/web/src/components/DocumentView.js

```diff
@@ -76,8 +76,10 @@ class DocumentView extends React.Component {
     if (this.state.manual) {
       this.setState(prevState => ({
         manual: {
-          ...prevState.manual,
-          texts: [...prevState.manual.texts, ''],
+		  content: {
+            ...prevState.manual.content,
+            texts: [...prevState.manual.content.texts, ''],
```

Comment:
indentation seems a little wild here

Reply:
(Seems you've mixed tabs and spaces a bit.)

Reply:
Indentation is all over the place in all the `.js` files. I'll see if I can apply an automatic, global fix